components/dashboard/src/teams/Members.tsx

```diff
@@ -209,7 +209,11 @@ export default function MembersPage() {
                         </ItemField>
                     </Item>
                     {filteredMembers.length === 0 ? (
-                        <p className="pt-16 text-center">No members found</p>
+                        <div className="flex h-screen items-center justify-center">
+                            <div className="h-16 w-16 animate-spin rounded-full border-t-2 border-b-2 border-gray-200">
+                                {/* Spinner */}
```

Comment:
**issue**: Did you intent to use the [`Loader`](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/components/Loader.tsx) component here? 😁

Reply:
@gtsiolis my bad 😅
Didn't realize that there was a standard component for that

I have added the `SpinnerLoader` instead of `SpinnerOverlayLoader` as the latter blocks the entire page which seems redundant